ddl/ddl_worker.go

```diff
@@ -282,41 +293,56 @@ func (w *worker) finishDDLJob(t *meta.Meta, job *model.Job) (err error) {
 	return errors.Trace(err)
 }
 
+func isDependencyJobDone(t *meta.Meta, job *model.Job) (bool, error) {
+	if job.DependencyID == 0 {
+		return true, nil
+	}
+
+	historyJob, err := t.GetHistoryDDLJob(job.DependencyID)
```

Comment:
Will `t.GetHistoryDDLJob` select the right job queue?

Reply:
Yes, it set key at https://github.com/pingcap/tidb/pull/6955/files#diff-625e9ab73a4a85a4a64f659183d95f3dR336.